<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultDisplay</class>
 <widget class="QWidget" name="ResultDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare Screen</string>
  </property>
  <widget class="QLabel" name="label_missingFolderNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_missingFilesNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_missingFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Missing Folder:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_missingFiles">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Missing Files:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_missingFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_missingFiles">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_copyFolder_all">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Will create all missing Folders(without the files) in the new directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Copy all Folder</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_copyAllFiles">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy all Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_properties">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
